### PR TITLE
Fix WPML compatibility

### PR DESCRIPTION
**Pull request by sc0ttkclark**

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
@srdjan-jcc is this officially all good to go now?

**Comment by srdjan-jcc** [comment]
@sc0ttkclark It's ready now.
Thank you.

**Comment by sc0ttkclark** [comment]
Thanks!

File: classes/PodsAPI.php [hunk 1]
```diff
@@ -7628,8 +7640,12 @@ public function get_table_info ( $object_type, $object, $name = null, $pod = nul
 			    'tt.taxonomy' => '`tt`.`' . $info[ 'field_type' ] . '` = "' . $taxonomy . '"'
 		    );
 
+            /*
+             * @todo wpml-comp WPML API call for is_translated_taxononomy
+             * @todo wpml-comp Check if WPML filters can be applied afterwards
+             */
 		    // WPML Support
-		    if ( is_object( $sitepress ) && !empty( $current_language ) && $sitepress->is_translated_taxonomy( $taxonomy ) && !$icl_adjust_id_url_filter_off ) {
+		    if ( is_object( $sitepress ) && !empty( $current_language ) && $sitepress->is_translated_taxonomy( $taxonomy ) && apply_filters( 'wpml_setting', true, 'auto_adjust_ids' ) ) {
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Is there not a taxonomy version of `wpml_is_translated_post_type` filter here?

File: classes/PodsAPI.php [hunk 2]
```diff
@@ -8343,6 +8361,11 @@ public static function get_current_language() {
 				} elseif ( isset( $current_screen->base ) && ( $current_screen->base == 'term' || $current_screen->base == 'edit-tags' ) ) {
 					// @todo MAYBE: Similar function like get_post_type for taxonomies so we don't need to check for $_GET['taxonomy']
 					if ( ! empty( $_GET['taxonomy'] ) ) {
+					    /*
+					     * @todo wpml-comp API call for taxonomy needed!
+					     * Suggested API call:
+					     * add_filter( 'wpml_is_translated_taxonomy', $_GET['taxonomy'], 10, 2 );
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: It sounds like we should be using apply_filters() here, right?

File: classes/PodsAPI.php (hunk 1 above)
Comment (srdjan-jcc) [inline classes/PodsAPI.php]: Not yet :( I opened API call request for WPML.

File: classes/PodsAPI.php (hunk 2 above)
Comment (srdjan-jcc) [inline classes/PodsAPI.php]: I added that as reminder. Should be switched to filter when available.

File: classes/PodsAPI.php (hunk 1 above)
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Well we can only hope :)

So this works as it is for now?

**Pull request merged**